### PR TITLE
feat : Playlists

### PR DESCRIPTION
**Pull request by TejasShekar**
- added functionality for managing playlists
- single playlist page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -39,6 +40,7 @@ function App() {
         <Route element={<RequireAuth />}>
           <Route path="/watch-later" element={<WatchLater />} />
           <Route path="/playlist" element={<Playlist />} />
+          <Route path="playlist/:playlistId" element={<SinglePlaylist />} />
           <Route path="/history" element={<History />} />
           <Route path="/liked" element={<LikedVideos />} />
```
Comment (ashishkatiyar2404) [inline src/App.js:45]: later u can add requireAuth for private pages

**Review by ashishkatiyar2404 — APPROVED** [review]
Suggested a change.
you can add a deploy link also for a better review.
your code functionality looks good to me. you can merge now

**Comment by anjali1102** [comment]
you handled empty input create new playlist. You added create playlist from /playlist route. liked the dynamic browser tab title on route change. nice, good work !

**Review by anjali1102 — APPROVED** [review]

**Comment by TejasShekar** [comment]
> Suggested a change.
> you can add a deploy link also for a better review.
> your code functionality looks good to me. you can merge now

Vercel bot adds the deploy link automatically. It's there. Just checked.

**Pull request merged**